pkg/envoy/rds/new_response_test.go

```diff
@@ -34,6 +34,8 @@ func TestNewResponse(t *testing.T) {
 	certSerialNumber := certificate.SerialNumber("123456")
 	testProxy := envoy.NewProxy(certCommonName, certSerialNumber, nil)
 
+	testOutbound := []*trafficpolicy.OutboundTrafficPolicy{}
```

Comment:
we should add a test case here that has some outbound traffic policy and check in the assertions that the correct virtual host is being programmed